docs/rfcs/002-atom-nightly-releases.md

```diff
@@ -0,0 +1,55 @@
+
+Today, a bleeding-edge user must manually pull Atom's `master` branch and compile their own build.  There is a source of `dev` builds from `master` across our CI services but those aren't made available to users as an official distribution.
+
+## Explanation
+
+A user who wants to use the latest improvements to Atom each day can go to atom.io, download the Atom Nightly release, and install it on their machine.  This release can be installed alongside Atom Stable and Atom Beta.
+
+Each night when there are new commits to Atom's `master` branch, a scheduled CI build creates a new Atom Nightly release with packages for Windows, macOS, and Linux.  These packages are automatically uploaded to a new GitHub release on the `atom/atom-nightly` repository using a nightly version based off of the current `dev` version in `master` (e.g. v1.29.0-dev.1 or v1.29.0-dev.20180601).
+
+Every 6 hours, an Atom Nightly release installed on Windows or macOS checks for a new update by consulting Electron's [update.electronjs.org](update-electron) service.  If a new update is available, it is downloaded in the background and the user is notified to restart Atom once it's complete.  This update flow is the same as what users experience in Atom Stable or Beta releases but occurs more frequently.
+
+Linux users must manually download nightly releases for now as there isn't an easy way to automatically install new updates across the various Linux distrubutions.  We may consider providing updatable [AppImage](http://appimage.org/) packages in the future; this will be proposed in a separate RFC.
+
+## Drawbacks
+
+There isn't a major downside to this effort since it would run in parallel to the existing Atom release process without affecting it.
+
+## Rationale and alternatives
+
+This is a useful approach because it allows us to achieve a much more rapid feedback loop with highly engaged users to ensure that Atom is improving regularly.  It's the best approach because it allows us to get rapid feedback without sacrificing the stability of the Stable and Beta releases.
+
+Another option is to speed up Atom's release cadence to ship Stable and Beta every two weeks (or more regularly).  This approach could shorten our feedback loop but at the expense of greater instability since new improvements would not have as much time to be polished before release.
+
+The impact of not taking this approach is that we continue to have to wait 1-2 months to get feedback from users about new features or bugs in Stable and Beta releases.
+
+## Unresolved questions
+
+- **What should we call this release channel?**
+
+  Some ideas:
+
+  - Atom Nightly
+  - Atom Reactor
+  - Atom Dev - Currently the name of dev builds but it might make sense to leave that for "normal" builds from `master`
```

Comment:
How about "Atom Canary"?

Other than that, I like "Atom Nightly." Descriptive and unambiguous.

Reply:
"Canary" is pretty common but IMO it doesn't sound cool enough :)  I want this channel to be enticing for users who wouldn't normally install bleeding-edge builds.  VS Code calls theirs "Insiders" which gives it more of an in-crowd feeling.  I like "Reactor" for this but I'm still looking for other names that might be better.

Now that I've thought about it some more, I don't like "Nightly" as much because it implies that there will be a new release every night.  Since we won't have new commits to master every single day, I worry that this will set an incorrect expectation.

Any other thoughts for cool-sounding channel names aside from "Reactor"?

Reply:
What about `Atom Alpha`? Might be more clear that it's about the version. `Atom Reactor` could be anything and is harder to guess.

Reply:
Alpha could work too, I definitely see your point.  Trying to resist the urge to do a Twitter poll :)

Reply:
If in doubt -> Twitter poll! 👍 :smile:

Reply:
I think I'll do one tomorrow morning for maximum exposure 😎